packages/components/src/styles/link-button-styles.ts

```diff
@@ -129,21 +130,7 @@ export const getLinkButtonStyles = (
           },
         })),
     },
-    label: {
-      ...buildResponsiveStyles(hideLabel, (hideLabelValue: boolean) =>
-        hideLabelValue
-          ? {
-              width: 0,
-              height: '1px',
-              textIndent: '-999999px',
-            }
-          : {
-              width: 'auto',
-              height: 'auto',
-              textIndent: 0,
-            }
-      ),
-    },
+    label: buildResponsiveStyles(hideLabel, (isHidden: boolean) => getHiddenTextJssStyle(isHidden)),
```

Comment:
Useless anonymous function:
```suggestion
    label: buildResponsiveStyles(hideLabel, getHiddenTextJssStyle),
```